src/ops.rs

```diff
@@ -59,6 +59,65 @@ impl Default for OpType {
     }
 }
 
+impl OpType {
+    /// The edge kind for the non-dataflow or constant-input ports of the
+    /// operation, not described by the signature.
+    ///
+    /// If not None, a single extra multiport of that kind will be present on
+    /// the given direction.
+    pub fn other_port(&self, dir: Direction) -> Option<EdgeKind> {
+        match dir {
+            Direction::Incoming => self.other_input(),
+            Direction::Outgoing => self.other_output(),
```

Comment:
seems worth replacing `other_input` and `other_output` with a function parametrized over Direction too?

Reply:
Many `OpTrait` implementations only override the output, so putting the parametrized function in the trait would just copy the `match` there.

I'm ok with either option.

Reply:
fair enough, happy to leave as is